app/assets/javascripts/logs.coffee

```diff
@@ -2,19 +2,21 @@ class Log
   logForm: ->
     enableMultiselect({placeholder: "Project Tags"})
     
-    #Ajax Post to save page
+    # Ajax Post to save page
     form = $("form.autosave")
     indicator = $(".auto-save-indicator")
-    form.on "change", ":input", ->
+    $('.autosave_link').on 'click', (event) ->
```

Comment:
Let's change these selectors to something more applicable since it's not an autosave feature anymore

```
".auto-save-indicator" => ".save-draft-indicator"
".autosave_link"       => ".save-draft-link"